### PR TITLE
Update attack-surface-reduction-exploit-guard.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3620
Added example query.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@levinec Review needed for a PR contributed by one of our freelance authors that addresses issue #3620. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md [hunk 1]
```diff
@@ -45,6 +45,19 @@ Triggered rules display a notification on the device. You can [customize the not
 
 For information about configuring attack surface reduction rules, see [Enable attack surface reduction rules](enable-attack-surface-reduction.md).
 
+## Review attack surface reduction events in the Windows Defender ATP Security Center
+
+Windows Defender ATP provides detailed reporting into events and blocks as part of its alert investigation scenarios.
+
+You can query Microsoft Defender ATP data by using [Advanced hunting](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/advanced-hunting-windows-defender-advanced-threat-protection). If you're using [audit mode](audit-windows-defender-exploit-guard.md), you can use Advanced hunting to see how controlled folder access settings would affect your environment if they were enabled.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]:
```suggestion
You can query Microsoft Defender ATP data by using [Advanced hunting](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/advanced-hunting-windows-defender-advanced-threat-protection). If you're using [audit mode](audit-windows-defender-exploit-guard.md), you can use Advanced hunting to see how controlled folder access settings could affect your environment.
```

File: windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md [hunk 2]
```diff
@@ -45,6 +45,19 @@ Triggered rules display a notification on the device. You can [customize the not
 
 For information about configuring attack surface reduction rules, see [Enable attack surface reduction rules](enable-attack-surface-reduction.md).
 
+## Review attack surface reduction events in the Microsoft Defender ATP Security Center
```
Comment (levinec) [inline windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: replace with "Microsoft Security Center"

Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]:
```suggestion
## Review attack surface reduction events in the Microsoft Security Center
```

**Comment by e0i** [comment]
@lindspea Please follow-up with requested changes. Thanks.

**Comment by e0i** [comment]
@levinec It looks like the requested changes were suggested by @nenonix and have been adressed. Please follow-up with merge if appropriate. Thanks.

**Pull request merged**